rules-tests/CodeQuality/Rector/ClassMethod/LocallyCalledStaticMethodToNonStaticRector/Fixture/skip_static_call_from_static.php.inc

```diff
@@ -11,8 +11,28 @@ final class SkipPrivateStaticCallFromStatic
         return static::reallyClear($input);
     }
 
+    public static function foo(string $input)
+    {
+        return self::reallyFoo($input);
+    }
+
+    public static function bar(string $input)
+    {
+        return SkipPrivateStaticCallFromStatic::reallyBar($input);
+    }
+
     private static function reallyClear($input)
     {
         return $input . ' - clean';
     }
+
+    private static function reallyFoo($input)
+    {
+        return $input . ' - foo';
+    }
+
+    private static function reallyBar($input)
+    {
+        return $input . ' - bar';
+    }
```

Comment:
Could you extract this one into a new test fixture file? Just so we have git history fixture tight with a specific fix

Reply:
Done, I've left the test case for self as I did not change anything for it, it was just missing